packages/flutter_tools/lib/src/test/test_compiler.dart

```diff
@@ -183,8 +182,8 @@ class TestCompiler {
           fileSystem: globals.fs,
           buildRunner: buildRunner,
         );
-      } else if (globals.platform.isLinux) {
-        (nativeAssetsYaml, _) = await buildNativeAssetsLinux(
+      } else if (globals.platform.isLinux || globals.platform.isWindows) {
+        (nativeAssetsYaml, _) = await buildNativeAssetsLinuxWindows(
```

Comment:
Can we split this into Linux and Windows cases that call the platform-specific wrappers since the fact that they are currently the same is an implementation detail?